docker/docker_image_dest.go

```diff
@@ -363,6 +365,15 @@ func (d *dockerImageDestination) TryReusingBlobWithOptions(ctx context.Context,
 			remoteName:   reference.Path(candidateRepo),
 			actions:      "pull",
 		}
+		compressionOperation, compressionAlgorithm, err := blobinfocache.OperationAndAlgorithmForCompressor(candidate.CompressorName)
+		if err != nil {
+			logrus.Debugf("... Failed: %v", err)
```

Comment:
(Non-blocking, this is pre-existing: “… failed” does not add much, and it is especially unhelpful if there are multiple paths where exactly the same text is used. `OperationAndAlgorithmForCompressor failed`, or something like that, would make a difference.)

Reply:
Done.